btcjson/walletsvrcmds_test.go

```diff
@@ -241,13 +241,27 @@ func TestWalletSvrCmds(t *testing.T) {
 				return btcjson.NewCmd("getnewaddress", "acct")
 			},
 			staticCmd: func() interface{} {
-				return btcjson.NewGetNewAddressCmd(btcjson.String("acct"))
+				return btcjson.NewGetNewAddressCmd(btcjson.String("acct"), nil)
 			},
 			marshalled: `{"jsonrpc":"1.0","method":"getnewaddress","params":["acct"],"id":1}`,
 			unmarshalled: &btcjson.GetNewAddressCmd{
 				Account: btcjson.String("acct"),
 			},
 		},
+		{
+			name: "getnewaddress optional addressType",
+			newCmd: func() (interface{}, error) {
+				return btcjson.NewCmd("getnewaddress", "acct", "bech32")
+			},
+			staticCmd: func() interface{} {
+				return btcjson.NewGetNewAddressCmd(btcjson.String("acct"), btcjson.String("bech32"))
```

Comment:
Can you please add another test for the case when the label is `nil` and the address type is specified?

```go
btcjson.NewGetNewAddressCmd(nil, btcjson.String("bech32"))
```